src/fields/Textarea.tsx

```diff
@@ -1,28 +1,25 @@
-import { useCallback, useMemo, useRef } from 'react'
+import { BaseSyntheticEvent, useCallback, useMemo } from 'react'
 import { Input } from 'rsuite'
 import styled from 'styled-components'
 
 import { Field } from '../elements/Field'
 import { FieldError } from '../elements/FieldError'
 import { Label } from '../elements/Label'
 import { useFieldUndefineEffect } from '../hooks/useFieldUndefineEffect'
-import { useKey } from '../hooks/useKey'
 import { normalizeString } from '../utils/normalizeString'
 
-import type { MutableRefObject, TextareaHTMLAttributes } from 'react'
+import type { InputProps } from 'rsuite'
 import type { Promisable } from 'type-fest'
 
-export type TextareaProps = Omit<
-  TextareaHTMLAttributes<HTMLTextAreaElement>,
-  'defaultValue' | 'id' | 'onChange' | 'value'
-> & {
+export type TextareaProps = Omit<InputProps, 'defaultValue' | 'id' | 'onChange' | 'value'> & {
   error?: string | undefined
   isLabelHidden?: boolean | undefined
   isLight?: boolean | undefined
   isUndefinedWhenDisabled?: boolean | undefined
   label: string
   name: string
   onChange?: ((nextValue: string | undefined) => Promisable<void>) | undefined
+  rows?: number
```

Comment:
Rather than doing that, we should include all Textarea props with an `InputProps & Omit<TextareaHTMLAttributes<HTMLTextAreaElement>, ...>`.

Reply:
Why? Expected props are the one used by `Input` component from `rSuite`. I understand typing with `TextareaHTMLAttributes<HTMLTextAreaElement>` is closer to what we expect from the component, but regarding what we said during the seminar, I don't think the tradeoff with legibility is worth it.